commands/faq.js

```diff
@@ -0,0 +1,14 @@
+const faq = require('../data/faq.json');
+
+module.exports.execute = async (client, message) => {
+	//const channel = message.channel();
+	//message.channel.send(faq['raid-room']);
```

Comment:
Can you throw these additions into a separate PR? Horace will send this in !help and don't want users confused about a non-existent command

Reply:
Should be good, couldn't figure this out until i realized i could just remove the file and send another commit 🤦‍♂️